docs/description.md

```diff
@@ -138,6 +138,9 @@ Missing variables with no default will produce an error.
 server:
   port: ${PORT:8001}
 ```
+#### Customizing low level parameters
```

Comment:
I'd move this into Local LLM Requirements sections, specifically before OSX GPU support. I think it fits better. Thanks